linode/manager
# upcoming: [M3-7702] - Update Region Select for edge sites

## Description 📝
Update our Region Select to support edge regions for Project Gecko

## Changes  🔄
- New `site_type` property to the Region type
- New `gecko` feature flag
- New icon to indicate an Edge region and updates to support displaying it

## Preview 📷
![image](https://github.com/linode/manager/assets/115299789/9b8cf181-709f-4470-b4be-e650c92d1e5a)

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Turn on the MSW and Gecko feature flag in DevTools 

### Verification steps
(How to verify changes)
- Go to `/linodes/create` and open the Region Select dropdown. You should see an icon next to the `Gecko Edge Test` region and also icon helper text next to the region select.
- You should _not_ see the icon or icon helper text for Marketplace, Cloning, or Images

```
yarn test RegionSelect
yarn test RegionSelect.utils
```

## As an Author I have considered 🤔

*Check all that apply*
